chapters/es/_toctree.yml

```diff
@@ -26,3 +26,23 @@
   - local: chapter1/10
     title: Quiz de final de capítulo
     quiz: 1
+
+- title: 2. Usando Transformers 🤗
+  sections:
+  - local: chapter2/1
```

Comment:
We should only keep the entries for the content that has been translated. So in this case, we'd need to remove all entries except those for `chapter2/4` and `chapter2/5`

Reply:
Hey @munozariasjm just checking to see if you'd like me to fix up the table of contents here since the rest of the PR looks good :)

Reply:
Yes!, if you could that would be great, thanks!!!